lib/workers/branch/lock-files.js

```diff
@@ -161,6 +189,28 @@ async function writeExistingFiles(config) {
   }
 }
 
+function listLocalLibs(dependencies) {
+  logger.trace(`listLocalLibs (${dependencies})`);
+  const toCopy = [];
+  for (const libName in dependencies) {
+    if ({}.hasOwnProperty.call(dependencies, libName)) {
+      const libVersion = dependencies[libName];
```

Comment:
Would `for (const [libName, libVersion] of Object.entries(dependencies)) {` work here instead?

Reply:
Yes it's working, and updated 👍